### PR TITLE
add infra command

### PR DESCRIPTION
**Pull request by mthenw**
I will do that in small steps. This is bare `infra` command extracted from `add-init` branch. As there is no TF -> apex integration yet I didn't include env switcher.

### CODE REVIEW

**Comment by mthenw** [comment]
@tj review plox

**Comment by tj** [comment]
LGTM but I think we still need the notion of an env, but works for now!

File: cmd/apex/infra/infra.go [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+package infra
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/apex/apex/cmd/apex/root"
+	"github.com/apex/apex/infra"
+)
+
+var infraEnvironment string
```
Comment (tj) [inline cmd/apex/infra/infra.go]: think this is dead code

Comment (mthenw) [inline cmd/apex/infra/infra.go]: damn!

**Comment by mthenw** [comment]
> LGTM but I think we still need the notion of an env, but works for now!

yeah but can we support env for infra but not for functions?

**Comment by tj** [comment]
I think we'll need it globally, for example to swap env variables for staging and prod, that's something I'll need in every app so I think a global notion would be good

**Comment by mthenw** [comment]
absolutely, I was asking more if we can have env flag only for infra in current moment.

**Comment by tj** [comment]
ohhh yea for sure

**Pull request merged**